### ISSUE TITLE
Compile-error/warning in ADBMobile.h

### ISSUE THREAD

**Issue by thomas3687** [Adobe-Marketing-Cloud/mobile-services#213]
The class method:
`+ (NSString * _Nonnull) version `
throws a compile-error/warning:

```
error: conflicting return type in declaration of 'version': 'NSInteger' (aka 'int') vs 'NSString * _Nonnull' [-Werror,-Woverriding-method-mismatch]
+ (nonnull NSString *) version;
^
```

```
/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator10.2.sdk/System/Library/Frameworks/Foundation.framework/Headers/NSObject.h:54:1: note: previous declaration is here
+ (NSInteger)version;
```

**Comment by sbenedicadb** [comment]
Hey @thomas3687,

We have created a ticket internally to resolve this warning/error.  I'll update this issue once we have released a resolution.  Thanks.

**steve benedick**
*sr mobile software engineer*
[Adobe Marketing Cloud](http://www.adobe.com/solutions/digital-marketing.html)

**Comment by sbenedicadb** [comment]
Hey @thomas3687,

This ticket has been resolved in the version released today, v4.13.6.  Thanks.

**steve benedick**
*sr mobile software engineer*
[Adobe Marketing Cloud](http://www.adobe.com/solutions/digital-marketing.html)
